edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Http/middleware/TlsConnectionFeatureExtended.cs

```diff
@@ -0,0 +1,14 @@
+//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Devices.Edge.Hub.Http.Middleware
+{
+    using System.Collections.Generic;
+    using System.Security.Cryptography.X509Certificates;
+
+    public class TlsConnectionFeatureExtended : ITlsConnectionFeatureExtended
```

Comment:
I would like to understand this a bit more.. where is the chain added to the class? If this is expected to be just a holder class, maybe the interface is not needed?

Reply:
Looks like this has no ref..